xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -6545,6 +6545,17 @@ glusterd_get_sock_from_brick_pid(int pid, char *sockpath, size_t len)
         ret = 0;
     }
 
+    /*
+     * Condition to check if the brick socket file is present
+     * in the stated path or not. This helps in preventing
+     * constant re-connect triggered in the RPC layer and also
+     * a log message would help out the user.
+     */
+    ret = sys_access(sockpath, F_OK);
```

Comment:
Kindly move this condition while tmpsockpath[0] is valid.

Reply:
ack.